src/Summoner/Template.hs

```diff
@@ -535,6 +535,8 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
             if [ -z "$$STACK_YAML" ]; then
               export PATH="/opt/ghc/$$GHCVER/bin:/opt/cabal/$$CABALVER/bin:$$PATH"
               echo $$PATH
+              echo "package $PKGNAME" >> cabal.project
```

Comment:
Yes, you're right. You need to escape `$` as it's reserved for vars in `neat-interpolation` – so you should use double `$`.

Reply:
Also, I am now thinking that if a user already has `cabal.project` in the repo to make it work together we need to add an empty line before "package $$PKGNAME"

Reply:
Also, instead of `$PKGNAME` you should insert `$repo` because we don't set `PKGNAME` environment variable in `.travis.yml`